tsconfig.packages.json

```
@@ -0,0 +1,21 @@
+{
+  "compilerOptions": {
+    "composite": true,
+    "esModuleInterop": true,
+    "lib": ["ES2020", "DOM"],
```

The extension still supports browser versions that don't support certain ES2020 features.
```suggestion
    "lib": ["ES2017", "DOM"],
```

---

Thanks! Changed in [0691206](https://github.com/MetaMask/controllers/pull/831/commits/06912063bf4f91fd58d87c7130a41defa4c88e0f).